pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -12,10 +12,10 @@ $border-radius: 4px;
 // Start styles
 html {
   scroll-behavior: smooth;
+  font-size: unset !important;
```

Comment:
This is where I overrode the `10px` html font-size — now it's using the default browser font-size which is `16px`.

Reply:
Just out of curiosity, why `unset` instead of `100%` ?
It's awesome that it works and I'm open to both variants, but I'm thinking of can it be confusing for someone first seeing it in the future?

Reply:
@levadadenys oh it does work with `100%`! It wasn't overriding it before, but I think that's when I still set the font-size on the `body` element. I'll change it 🙌